packages/account/src/Sections/Security/Passkeys/components/passkeys-status.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
+import FormBody from '../../../../Components/form-body';
+
+type TPasskeysStatus = {
+    title: React.ReactElement;
+    description?: React.ReactNode;
+    icon: string;
+    className?: string;
+};
+
+const PasskeysStatus = ({
+    title,
+    description,
+    icon,
+    children,
+    className,
+}: React.PropsWithChildren<TPasskeysStatus>) => {
+    return (
+        <React.Fragment>
+            <FormBody
+                scroll_offset='22rem'
+                className={classNames('passkeys-status__wrapper', {
+                    [`${className}`]: className,
```

Comment:
What's the purpose of this [`${className}`]?

Reply:
to add classname if it is passed via props

Reply:
then it should be done like this instead :) :
className={classNames('passkeys-status__wrapper', className)}

Reply:
but in this case we will see in classname list following: 'passkeys-status__wrapper undefined',
if don't pass any value.
I believe we should add optional classnames in this way.
there is another example: DataTable in components.

Reply:
@maryia-deriv , @suisin-deriv , you are team. the library handles such case. I have confused with some other similar case.
removing this